browse/controllers/years_operating.py

```diff
@@ -0,0 +1,41 @@
+"""Year link functions."""
+
+from typing import Dict, Any, Tuple, List, no_type_check
+from datetime import date
+
+from flask import url_for
+
+
+def years_operating(archive: Dict[str, Any]) -> List[int]:
```

Comment:
This might work better as a utility function in `arxiv.taxonomy`?

Reply:
I'd leave it here. The ordering is display related. 

It is split out because it is used by both the /archive page and the /year page.